config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
   root to: 'application#index'
 
+  # require 'sidekiq'
   require 'sidekiq/web'
   mount Sidekiq::Web, at: 'sidekiq'
+
+  # Sidekiq::Web.use Rack::Auth::Basic do |username, password|
```

Comment:
Why are we commenting this ? Don't we want to restrict access always ?

Reply:
it was commented at config.ru so i thought that it has to be commented in here too, but i'll uncomment it